src/styles/profit_and_loss_detailed_charts.scss

```diff
@@ -160,6 +160,7 @@ header.Layer__profit-and-loss-detailed-charts__header--tablet {
       white-space: nowrap;
       overflow: hidden;
       text-overflow: ellipsis;
+      z-index: 999;
```

Comment:
Does this cause it to clip the project select dropdown? That was causing problems for me earlier.

Reply:
If so, z index 1 might be sufficient

Reply:
This was speculative because I can't actually replicate the issue justin is having, so I'll just do 1 for now